lib/options.js

```diff
@@ -20,6 +19,16 @@ function override(options, overrides) {
   }
 }
 
+function getShareAbleConfigPath(options) {
+  if (!options || !options.extends) {
```

Comment:
I can only see line 48 calling this in which case `options.extends` exists. Is there another call that I'm missing?

Reply:
No, it's the only call. I will remove the function.

Reply:
Fixed